Getting-Started-with-Akka-http-signature.md

```diff
@@ -283,24 +283,38 @@ To demonstrate how access control changes user permitted actions, one can also m
 
 #   acl:agent [ cert:key <publicKey#key> ];
 
-	acl:agentClass <http://xmlns.com/foaf/0.1/Agent> ;
+#	acl:agentClass <http://xmlns.com/foaf/0.1/Agent> ;
 
 	acl:mode acl:Read .
 
-
-
 <> acl:include <.acl> .
 ```
 
 This allows the user to manipulate access control by simply commenting out `acl:agent [ cert:key <publicKey#key> ];` 
 
 After that, the access to the Key files specified will be restricted. And therefore they will not be usable via the symbolic link created earlier. 
 
-If one attempts to do so, via curl for example, they will be presented with the following error message: 
+If one now restarts the server and attempts to access the files, via curl for example, they will be presented with the following error message: 
 
 ```bash
 $ curl -i -k https://localhost:8443/2013/pubKey.acl
 
-curl: (7) Couldn't connect to server
+HTTP/1.1 404 Not Found
+Content-Type: text/plain; charset=utf-8
```

Comment:
that is not the error code we are looking for. We were expecting an `Authentacation Required` error code. You are currently only getting a problem because you have probably forgot to set up the symbolic link. Instead of doing all of this, I would just add the rdf to the existing `key` file. That will make things a bit simpler.